### PR TITLE
Technical Review: tshoot-registerclones.adoc

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/9518

### CODE REVIEW

File: modules/administration/pages/tshoot-registerclones.adoc [hunk 1]
```diff
@@ -19,17 +19,25 @@ If more detailed instructions are required, put them in a "Resolving" procedure:
 . Last step
 ////
 
-Sometimes a cloned client (either traditional or Salt) will use the same machine ID as the system they are a clone of.
-This results in {productname} only recognizing one system, rather than two different systems.
-This can be resolved by changing the machine ID of the  cloned system, so that {productname} recognizes them as two different clients.
+
+If you are using {productname} to manage virtual machines, you might find it useful to create clones of your VMs.
+A clone is a VM that uses a primary data volume that is an exact copy of an existing data volume.
```
Comment (keichwa) [inline modules/administration/pages/tshoot-registerclones.adoc]: I woudn't call this a data volume (ok, everything is data...).  You primiliary clone the volume with the system data (the OS).
If you agree we must change the surrounding text, too.

File: modules/administration/pages/tshoot-registerclones.adoc [hunk 2]
```diff
@@ -19,17 +19,25 @@ If more detailed instructions are required, put them in a "Resolving" procedure:
...
-The cloned virtual machine must have a different UUID than the original (the UUID is generated by your hypervisor), otherwise {productname} will override the original system data with the new data.
+Each step of this procedure is performed on the cloned client.
+This procedure does not manipulate the original client, which will still be registered to {productname}.
 ====
 
+
+
 .Procedure: Resolving Duplicate Machine IDs in Cloned Salt Clients
 
 . For {slsa}{nbsp}12: If your machines have the same machine ID, delete the file on each client and re-create it:
```
Comment (keichwa) [inline modules/administration/pages/tshoot-registerclones.adoc]:
```suggestion
. For {slsa}{nbsp}12: If your machines have the same machine ID, delete the file on each duplicated client and re-create it:
```
just a proposal for clarity.

**Comment by Loquacity** [comment]
@juliogonzalez can you please approve when you get a chance?

**Comment by juliogonzalez** [comment]
Why me :-?

**Comment by admd** [comment]
> @juliogonzalez can you please approve when you get a chance?

You probably meant @moio

**Comment by Loquacity** [comment]
> > @juliogonzalez can you please approve when you get a chance?
> 
> You probably meant @moio

I think it autosuggested you @juliogonzalez, but yes, I meant @moio.

File: modules/administration/pages/tshoot-registerclones.adoc [hunk 3]
```diff
@@ -19,20 +19,28 @@ If more detailed instructions are required, put them in a "Resolving" procedure:
...
+Each step of this procedure is performed on the cloned client.
+This procedure does not manipulate the original client, which will still be registered to {productname}.
 ====
 
+
+
 .Procedure: Resolving Duplicate Machine IDs in Cloned Salt Clients
 
-. For {slsa}{nbsp}12: If your machines have the same machine ID, delete the file on each client and re-create it:
+. For {slsa}{nbsp}12: If your machines have the same machine ID, delete the file on each duplicated client and re-create it:
```
Comment (moio) [inline modules/administration/pages/tshoot-registerclones.adoc]: This is about SLE 12, 15, openSUSE, RHEL 7, 8, Ubuntus...

We should rather write "distributions that support systemd"

**Pull request merged**